### PR TITLE
Forseti Config Validator: initial integration

### PR DESCRIPTION
**Pull request by MartinPetkov**
I successfully ran the following commands from the [guide](https://google.github.io/oss-fuzz/getting-started/new-project-guide/#testing-locally):

```sh
python infra/helper.py build_image config-validator
python infra/helper.py build_fuzzers --sanitizer address config-validator
python infra/helper.py check_build config-validator
python infra/helper.py run_fuzzer config-validator fuzz_config_validator
```

### CODE REVIEW

**Comment by MartinPetkov** [comment]
@morgante I've added you as a CC FYI

File: projects/config-validator/project.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/GoogleCloudPlatform/config-validator/"
+main_repo: "https://github.com/GoogleCloudPlatform/config-validator/"
+primary_contact: "martin.p.petkov@gmail.com"
+auto_ccs:
+- morgante.pell@morgante.net
```
Comment (morgante) [inline projects/config-validator/project.yaml:5]: @MartinPetkov Can we use my work email instead? `morgantep@google.com`

Comment (MartinPetkov) [reply projects/config-validator/project.yaml]: Of course, done.

**Comment by MartinPetkov** [comment]
@jonathanmetzman @oliverchang @inferno-chromium I saw you were @-ed on other PRs, I'm doing so too. Let me know if this isn't the right way to request a review, I wasn't sure how to put this PR in the review queue.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**